messaging/config.go

```diff
@@ -3,14 +3,23 @@ package messaging
 import (
 	"github.com/microdevs/missy/service"
 	"strconv"
+	"time"
 )
 
 const defaultKafkaMaxRetries = 3
 const defaultKafkaRetriesIntervalMS = 5000
+const defaultKafkaRetentionTime = time.Minute * 60 * 24 * 30
+
+const (
+	kafkaRetriesMaxNumber  = "kafka.retries.max.number"
+	kafkaRetriesIntervalMS = "kafka.retries.interval.ms"
+	kafkaRetentionTime     = "kafka.retention.time"
+)
 
 // InitConfig registers the standard config for this package
 func InitConfig() {
-	service.Config().RegisterOptionalParameter("KAFKA_RETRIES_MAX_NUMBER", strconv.Itoa(defaultKafkaMaxRetries), "kafka.retries.max.number", "The number of times a kafka reader will retry")
-	service.Config().RegisterOptionalParameter("KAFKA_RETRIES_INTERVAL_MS", strconv.Itoa(defaultKafkaRetriesIntervalMS), "kafka.retries.interval.ms", "The time between retries in a kafka reader in ms")
+	service.Config().RegisterOptionalParameter("KAFKA_RETRIES_MAX_NUMBER", strconv.Itoa(defaultKafkaMaxRetries), kafkaRetriesMaxNumber, "The number of times a kafka reader will retry")
+	service.Config().RegisterOptionalParameter("KAFKA_RETRIES_INTERVAL_MS", strconv.Itoa(defaultKafkaRetriesIntervalMS), kafkaRetriesIntervalMS, "The time between retries in a kafka reader in ms")
+	service.Config().RegisterOptionalParameter("KAFKA_RETENTION_TIME", defaultKafkaRetentionTime.String(), kafkaRetentionTime, "Consumer retention duration on kafka broker, default 24h")
```

Comment:
You need to update the description of the config option: 

24h -> 30 days | maybe let's use here a constant